corehq/apps/data_analytics/tests/test_malt_generator.py

```diff
@@ -348,29 +348,58 @@ def setUp(self) -> None:
         self.web_user = create_user_for_malt_tests(is_web_user=True, user_id='user_id_1', username='user1')
         self.mobile_user = create_user_for_malt_tests(is_web_user=False, user_id='user_id_2', username='user2')
         self.app_data = create_malt_app_data()
+        self.users_by_id = {'user_id_1': self.web_user, 'user_id_2': self.mobile_user}
 
         self.patcher = patch('corehq.apps.data_analytics.malt_generator._get_malt_app_data')
         self.mock_get_malt_app_data = self.patcher.start()
         self.mock_get_malt_app_data.return_value = self.app_data
 
+        self.patcher2 = patch('corehq.apps.data_analytics.malt_generator.get_last_form_submission_received')
+        self.mock_get_last_form_submission = self.patcher2.start()
+        self.mock_get_last_form_submission.return_value = datetime.datetime(2022, 1, 10, 0, 0)
+
+        self.patcher3 = patch('corehq.apps.data_analytics.malt_generator.get_app_submission_breakdown_es')
```

Comment:
Nit: could create better names for these patchers, i.e. `submissions_received_patcher` and `breakdown_es_patcher`. Not huge, but its a bit cryptic to see `patcher`1-3

Reply:
See 80a39f63f862adc8bdea4bfc8cc3e1f3889082e1